Automattic/wp-calypso
# Enable image editing on private sites, don't use photon to load images

Enables image editing on private sites, removing all checks that were disabling it. With D58521-code applied, image requests to `*.files.wordpress.com` get the right CORS headers that allow the image to be read by scripts and loaded into a canvas.

Also removes usage of Photon URLs to load blog images. Let's load everything from `*.files.wordpress.com` for consistency sake. Photon is a CDN for site visitors, shouldn't be used by an admin tool.

**How to test:**
1. Apply D58521-code to your sandbox
2. Point the `yourprivatesite.files.wordpress.com` name to your sandbox
3. Open Calypso Media library for `yourprivatesite.files.wordpress.com`
4. Select an image and edit it. Rotate, crop, save back.

Editing a site icon in Site Settings should be also possible now.

Fixes #25115 🙃
